cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml

```diff
@@ -243,6 +243,8 @@ objects:
             done < /tmp/journals
           done < /tmp/nodes
 
+          gzip /tmp/artifacts/ansible.log
```

Comment:
This won’t work because if the build fails you’ll be uploading the uncompressed version, which defeats the purpose of gzipping.

Reply:
So the artifacts get uploaded after every container, not just teardown? Okay, put it in exit trap in setup container instead